src/models/selectedOption.ts

```diff
@@ -0,0 +1,4 @@
+export interface SelectedOption {
+  id: string;
+  name: string;
```

Comment:
External Reference Catalog Service, Used mainly by our Kamui Team, will not be compatible with that object. Their Option Object is like this:
``
export interface SelectedOption
{
id : string;
description : string;
}
``

Reply:
We had the same problem on our Dynamic View Engine Auto-Completes. For that reason we used an Untyped "Any" Object

Reply:
Hi @javierpardollama , you are right in your observation, however we can take advantage of this context to start normalizing this model for any application.

In any case, since we are still on beta versioning for this changes, we can just merge as it is and start analysing the impact in our apps (Remember that this is Cotecna business agnostic ;) )

Many thanks